src/prefect/states.py

```diff
@@ -495,13 +500,34 @@ def Pending(cls: Type[State] = State, **kwargs) -> State:
     return schemas.states.Pending(cls=cls, **kwargs)
 
 
-def Paused(cls: Type[State] = State, **kwargs) -> State:
+def Paused(
+    cls: Type[State] = State,
+    timeout_seconds: int = None,
+    pause_timeout: datetime.datetime = None,
```

Comment:
It's pretty hard to understand the difference between these without reading the implementation. Can we use clearer names?